faunadb-java-dsl/src/main/java/com/faunadb/client/types/Result.java

```diff
@@ -74,9 +74,11 @@ public String toString() {
   private static final class Failure<A> extends Result<A> {
 
     private final String error;
+    private final Throwable throwable;
```

Comment:
Can we make it optional? I know it's been used in a single place right now but, I feel like an option better express the multiplicity of this value.